docs/source/queries/prepared.md

```diff
@@ -1,8 +1,12 @@
 # Prepared query
 
-Prepared queries provide much better performance than simple queries,
+Prepared queries provide much better performance than unprepared queries,
 but they need to be prepared before use.
 
+There are other benefits that prepared queries have to offer:
+- Increased type-safety - thanks to metadata provided by the server, the driver can verify bound values' types before serialization. This way, we can be always sure that the Rust type provided by the user is compatible (and if not, the error is returned) with the destined native type. The same applies for deserialization.
```

Comment:
Unprepared statements are type safe too because they are prepared before execution. They return the same errors.
I'd argue they are safer because they don't suffer from the problem with schema changes - ser / deser will always use current metadata (unless schema is changed between preparation and execution)